docs/DeveloperGuide.md

```diff
@@ -1232,12 +1315,123 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. Nurse's input is invalid command format.
+
+  * 3a1. PatientSync shows an error message.
+
+    Use case resumes at step 2.
+
+* 3b. Nurse's input contains invalid patient index.
+
+  * 3b1. PatientSync shows an error message.
+
+    Use case resumes at step 2.
+
+* 3c. Nurse's input contains invalid tags.
+
+  * 3c1. PatientSync shows an error message.
+
+    Use case resumes at step 2.
+
```

Comment:
* 3d. The patient selected does not contain specified tag
* 3d1. PatientSync shows an error message.
Use case resumes at step 2

Reply:
Updated!